Libplanet/Store/TrieStateStore.cs

```diff
@@ -183,5 +167,35 @@ public void Dispose()
         /// <paramref name="blockHash"/>.</exception>
         public HashDigest<SHA256> GetRootHash(HashDigest<SHA256> blockHash)
             => new HashDigest<SHA256>(_stateHashKeyValueStore.Get(blockHash.ToByteArray()));
+
+        internal HashDigest<SHA256>? SetStates<T>(
+            Block<T> block,
+            IImmutableDictionary<string, IValue> states,
+            bool rehearsal)
```

Comment:
Do we need this rehearsal option? This seems to be only used when making the genesis block. Since the `MakeGenesisBlock` method uses an in-memory `DefaultKeyValueStore`, wouldn't it be okay without this option?

Reply:
You're right. I will remove it.

Reply:
I fixed it in 68090f7. Please check your intention was applied well.